pkg/host/host.go

```diff
@@ -20,7 +20,8 @@ func DetectSupportedSyscalls(target *prog.Target, sandbox string) (
 	supported := make(map[*prog.Syscall]bool)
 	unsupported := make(map[*prog.Syscall]string)
 	// Akaros does not have own host and parasitizes on some other OS.
```

Comment:
Should we update these comments?

Reply:
Probably. I'd kind of hate to update all the comments to just say "Akaros and Fuchsia blah blah" though. Seems like it ties into having a `pkg.IsHostFuzzOS` helper function like I suggested [above](https://github.com/google/syzkaller/pull/1350#issuecomment-522816038).